src/pkg/packager/common.go

```diff
@@ -352,3 +362,117 @@ func (p *Packager) handleIfPartialPkg() error {
 	p.cfg.DeployOpts.PackagePath = destination
 	return nil
 }
+
+func (p *Packager) validatePackageChecksums() error {
+
+	// Run pre-checks to make sure we have what we need to validate the checksums
+	_, err := os.Stat(p.tmp.Checksums)
+	if err != nil {
+		return fmt.Errorf("unable to validate checksums as we are unable to find checksums.txt file within the package")
+	}
+	if p.cfg.Pkg.Metadata.AggregateChecksum == "" {
+		return fmt.Errorf("unable to validate checksums because of missing metadata checksum signature")
+	}
+
+	// Get a glob of all the files within the package
+	packageGlob, err := filepath.Glob(filepath.Join(p.tmp.Base, "*"))
```

Comment:
This should be a filepath walk just like how the generate checksums does it - otherwise you could hide a file inside of a zarf package at a lower folder level.

Reply:
^ because filepath.Glob does not have a `**` feature, it will only glob everything that is at the base level of the tmp dir.